addons/website_blog/views/website_blog_templates.xml

```diff
@@ -586,7 +586,7 @@ list of filtered posts (by date or tag).
 <template id="blog_post_template_new_post">
     <p>A new post <t t-esc="post.name" /> has been published on the <t t-esc="object.name" /> blog. Click here to access the blog :</p>
     <p style="margin-left: 30px; margin-top: 10 px; margin-bottom: 10px;">
-        <a t-attf-href="/blog/#{slug(object)}/post/#{slug(post)}"
+        <a t-attf-href="#{object.get_base_url() or ''}/blog/#{slug(object)}/post/#{slug(post)}"
```

Comment:
Sadly in 13.0 we don't yet have the website.domain "cleaning" which is in charge of removing the trailing slash and adding a scheme.
In 14.0, `get_base_url()` will do that through the _get_http_domain which is smarter: https://github.com/odoo/odoo/commit/3ad775aab717b395a5d11527aeb3596af66afa99
In 15.0, `get_base_url()` will do that through the domain directly as the write/create will ensure the domain is always saved correctly: https://github.com/odoo/odoo/commit/042c95b0219bb0aa13e73385e092fa76ff1a1b0a

We can't join url like that as this in 13.0 as it will results to `https://15455099-13-0.runbot36.odoo.com//blog/astronomy-2/post/azeaeaze-10` where the double `//` will lead to 404 pages if the user didn't remove the trailing / when setting the domain.
It is a common use case as people sometimes simply copy paste their website URL from their browser URL bar which copies that trailing slash.

Here, you can probably do the `url_join` python side and send a new full link param to the template.
Then just verify that the new key exists and use it, else fallback to the current code (so if someone used that template in a custo but didn't ofc provide the new key it won't crash).

Also, `/blog/#{slug(object)}/post/#{slug(post)}` is just `post.website_url`